src/extensibility/node/package-validator.js

```diff
@@ -173,6 +173,15 @@ function findCommonPrefix(extractDir, callback) {
 }
 
 /**
+ * Strip the BOM from the file if it has one.
+ * @param {String} data The raw file contents.
+ * @return {String} The same data, sans the BOM.
+ */
+function stripBOM(data) {
+    return data.substr(data.indexOf("{")).replace(/\u0000/g, "");
```

Comment:
Would you prefer I reimplement that library or would it be better to import and `require` it? Just thinking in case it would have use in other places.